build/csslint/xfail.list

```diff
@@ -108,6 +108,7 @@ apps/system/style/modal_dialog/modal_dialog.css
 apps/system/style/notifications/notifications.css
 apps/system/style/software_button_manager/software_button_manager.css
 apps/system/style/sound_manager/sound_manager.css
+apps/system/style/statusbar/statusbar.css
```

Comment:
It still fails returning a RBRACE error. Couldn't find another solution... May be we can remove the ignore rule in the file, though